baseapp/testproject/graphql.py

```diff
@@ -26,6 +26,8 @@
 from baseapp_reactions.graphql.mutations import ReactionsMutations
 from baseapp_reactions.graphql.queries import ReactionsQueries
 from baseapp_reports.graphql.mutations import ReportsMutations
+from baseapp.content_feed.graphql.mutations import ContentFeedMutations
+from baseapp.content_feed.graphql.queries import ContentFeedQueries
```

Comment:
You imported but didn't use, you need to add them to `Query` and `Mutation` classes